.config-schema.json

```diff
@@ -250,14 +250,14 @@
                     "description": "the maximum amount of time a connection to the datastore may be idle",
                     "type": "string",
                     "format": "duration",
-                    "default": "connections are not closed due to idle time - database/sql default",
+                    "default": "",
                     "x-env-variable": "OPENFGA_DATASTORE_CONN_MAX_IDLE_TIME"
                 },
                 "connMaxLifetime": {
                     "description": "the maximum amount of time a connection to the datastore may be reused",
```

Comment:
```suggestion
                    "description": "the maximum amount of time a connection to the datastore may be reused. If zero, connections are not closed due to a connection's age.",
```

Reply:
Please also update the text that gets used when building the OpenFGA server: https://github.com/openfga/openfga/blob/cfd75567075ef916f6e871e8c8cf787e9170bbc7/cmd/run/run.go#L157-L159